airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -1492,7 +1483,7 @@ class Contacts(CRMSearchStream):
     last_modified_field = "lastmodifieddate"
     associations = ["contacts", "companies"]
     primary_key = "id"
-    scopes = {"crm.objects.contacts.read"}
+    scopes = {"crm.objects.contacts.read", "crm.objects.contacts.read"}
```

Comment:
@a-rampalli crm.objects.contacts.read this is added twice.

Reply:
Removing..